src/pages/Farm/index.tsx

```diff
@@ -130,8 +131,8 @@ const Farm = () => {
       .flat()
       .filter(
         item =>
-          (item.endTime && item.endTime > currentTimestamp) ||
-          (blockNumber && item.endBlock && item.endBlock > blockNumber),
+          (item.version === FairLaunchVersion.V2 && item.endTime > currentTimestamp) ||
```

Comment:
how about farm v3 here?

Reply:
all new farms will use timestamp, then we can remove item.version check here

Reply:
For now, we didn't have FarmV3 type yet, so it will be error when being used